Readme.md

```diff
@@ -120,18 +112,18 @@ re.exec('/bar/baz')
 //=> ['/bar/baz', 'bar/baz']
 ```
 
-#### Custom Match Parameters
+#### Custom Matching Parameters
 
-All parameters can be provided a custom regexp, which overrides the default (`[^\/]+`).
+All parameters can be provided a custom regexp, which overrides the default match (`[^\/]+`). For example, you can match digits in the path:
 
 ```js
-var re = pathToRegexp('/:foo(\\d+)')
+var re = pathToRegexp('/icon-:foo(\\d+).png')
 // keys = [{ name: 'foo', ... }]
 
-re.exec('/123')
-//=> ['/123', '123']
+re.exec('/icon-123.png')
+//=> ['/icon-123.png', '123', index: 0, input: '/icon-123.png']
```

Comment:
Going to remove the extra two params here since none of the other examples use it.